src/main/java/nl/ordina/jobcrawler/scrapers/JobBirdScraper.java

```diff
@@ -157,7 +181,7 @@ private boolean continueSearching(ArrayList<String> vacancyURLs, ArrayList<Strin
      * @param doc The HTML document containing the URLs to the vacancies
      * @return the index of the last page to scrape
      */
-    private int getLastPageToScrape(Document doc) {
+    protected int getLastPageToScrape(Document doc) throws HTMLStructureException {
         int totalNumberOfPages = getTotalNumberOfPages(doc);
         // TODO: we could get more sophisticated logic in place to limit the number of pages.
         // For example, we could look at the posting date of each vacancy, and limit it to thirty days.
```

Comment:
this method does not throw that exception, so I do not think it is necessary

Reply:
adapted